addon/components/polaris-drop-zone.js

```diff
@@ -404,13 +404,15 @@ export default Component.extend(ContextBoundEventListenersMixin, {
   adjustSize() {
     throttle(this, function() {
       let node = this.get('node');
-      let size = 'large';
+      let size = 'extraLarge';
```

Comment:
Maybe worth pulling this value into a `defaultSize` constant

Reply:
yeah, can do